smoketest/configs/bgp-big-as-cloud

```diff
@@ -1823,12 +1823,6 @@ system {
             }
             version 9
         }
-        sflow {
-            agent-address auto
-            server 1.2.3.4 {
-                port 1234
-            }
-        }
```

Comment:
Do not delete the config from the previous versions as they also checks migrations between versions.
This config from 1.2.5 should be migrated to the new syntax without issues.

Reply:
Good catch